lib/router/transition.js

```diff
@@ -16,7 +16,7 @@ import { trigger, slice, log, promiseLabel } from './utils';
   @param {Object} error
   @private
  */
-function Transition(router, intent, state, error) {
+function Transition(router, intent, state, error, previousTransition) {
```

Comment:
@alexspeller I spoke too soon, there is a closure to this context being created.